build.sh

```diff
@@ -103,6 +103,22 @@ for i in "${ARGS[@]}"; do
 	esac;
 done
 
+if ! command -v /bin/bear &> /dev/null; then
+	echo "'bear' executable not found. Compilation database will not be built"
+elif [ -z "${DO_NOT_EXEC_BEAR+x}" ]; then
+	b_log "Running bear on top of build script"
+
+	OUTPUT_FILE=_build/"$TARGET"/compile_commands.json
+
```

Comment:
maybe: `ln -sf  "$OUTPUT_FILE" ./compile_commands.json` to provide compilation database in project root for other tools to reuse it.

Also: if You do it, there is no need to pass `OUTPUT_FILE` to the bear directly (will be handled by symlink).

If You decide on doing the **global symlink** please add it to `.gitignore` and probably clean it up on `clean` command